oqs-template/generate.yml

```diff
@@ -582,37 +582,37 @@ sigs:
         name: 'mldsa44'
         pretty_name: 'ML-DSA-44'
         oqs_meth: 'OQS_SIG_alg_ml_dsa_44'
-        oid: '1.3.6.1.4.1.2.267.12.4.4'
-        code_point: '0xfed0'
+        oid: '2.16.840.1.101.3.4.3.17'
+        code_point: '0x0904'
```

Comment:
Please add comment(s) pointing to the (draft) RFCs at each OID or code point where available so "posterity" can understand/track things.

Reply:
added in [c958282](https://github.com/open-quantum-safe/oqs-provider/pull/568/commits/c9582825a049b5f957ec953b5a2c5155d8cd09f4)